layouts/base.html

```diff
@@ -114,6 +114,16 @@
     <div class="govuk-width-container">
       <div class="govuk-footer__meta">
         <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+          <div class="govuk-footer__contact">
+            <p class="govuk-footer__contact-title">Contact us for support</p>
+            <p class="govuk-footer__contact-description">
+              Contact us via this email:
+              <a href="mailto:school.workloadandwellbeing@education.gov.uk" class="govuk-link">school.workloadandwellbeing@education.gov.uk</a>,
```

Comment:
this link doesn't work for me, is it just because it's in a test enviro?

Reply:
spoke with mark and we're going to test this outside of deploy previews